lib/iris/tests/integration/plot/test_plot_2d_coords.py

```diff
@@ -26,10 +26,16 @@
 # importing anything else
 import iris.tests as tests
 
-import cartopy.crs as ccrs
+import numpy as np
```

Comment:
@pp-mo why is the `NumPy` import separate at the top? At minimum I feel it should be after the `pyplot` import. Preferably they would both be after the Iris imports.

Reply:
Presumably you mean "NumPy ... after the pyplot ... ***before*** the Iris imports" ??

My understanding of our guidelines for imports is to separate into 3 or 4 specific groups : 
  1. standard-library packages
  2. other 3rd party libraries
  3. "own" packages + modules
  4. (in tests only) test subjects

So here, numpy and pyplot should be in the same block, 
but before any Iris stuff 
(except iris.tests, which is required to happen first. :nauseated_face: yech ).

This agree with your understanding @dkillick ?

Reply:
> This agree with your understanding

Yes... and no, at the same time! Basically, I feel like we've slipped away from separating (2.) and (3.), even in test modules. It's not worth sweating about here, but I would still like to see the `NumPy` import at least after the `pyplot` import 😉

Reply:
:+1: